core/vm/eips.go

```diff
@@ -142,10 +142,9 @@ func enable2929(jt *JumpTable) {
 	jt[DELEGATECALL].constantGas = params.WarmStorageReadCostEIP2929
 	jt[DELEGATECALL].dynamicGas = gasDelegateCallEIP2929
 
-	// This was previously part of the dynamic cost, but we're using it as a constantGas
-	// factor here
-	jt[SELFDESTRUCT].constantGas = params.SelfdestructGasEIP150
-	jt[SELFDESTRUCT].dynamicGas = gasSelfdestructEIP2929
+	// SELFDESTRUCT is disabled in Scroll
```

Comment:
Why not delete it directly?

Reply:
No, we should not delete it in case we may support it in the future and it'll incur more incompatiblity.

Reply:
ok, that's great.